actionpack/lib/action_dispatch/middleware/stack.rb

```diff
@@ -5,6 +5,16 @@
 
 module ActionDispatch
   class MiddlewareStack
+    class FakeSendfile
```

Comment:
```suggestion
    class FakeSendfile # :nodoc:
```

This class should probably not be documented (see e9e8ea2b2071c8228625a72d51e0d3a589611eaa)